File: JavaScript/Clase-1/Ejercicios/Ejercicio-1.js
console.log("Ejercicio 1: Números divisibles entre 3 de 0 a 10");
for (let num = 0; num <= 10; num++) {
    if (num % 3 === 0) {
        process.stdout.write(num + ",");
    }
}
console.log("\n"); // Salto de línea

File: Python/Clase-1/Ejercicios/Ejercicio-1.py
# Ejercicio 1: Iterar un rango de 0 a 10 e imprimir números divisibles entre 3
print("Ejercicio 1: Números divisibles entre 3 de 0 a 10")
for num in range(0, 11):
    if num % 3 == 0:
        print(num, end=",")
print("\n")  # Salto de línea